modules/react/layout/lib/Grid.tsx

```diff
@@ -5,7 +5,7 @@ import {Box, BoxProps} from './Box';
 
 import {grid, GridStyleProps} from './utils/grid';
 
-export type GridProps = BoxProps & GridStyleProps;
+export type GridProps = Omit<BoxProps, 'display'> & GridStyleProps;
```

Comment:
Same as we did with Flex.

VSCode intellisense was getting confused with BoxProps display and not providing any helpful suggestions. So instead of merging Box's `display` with Grid's `display`, we're removing one and adding the other. The types remain the same, but VSCode works better.